triage/forms.py

```diff
@@ -109,12 +109,20 @@ def get_persona(cleaned_data):
     return NEW_EXPORTER
 
 
-def get_has_exported_before(cleaned_data):
-    return cleaned_data.get('exported_before') is True
+def get_has_exported_before(answers):
+    return answers.get('exported_before') is True
 
 
-def get_is_regular_exporter(cleaned_data):
-    return cleaned_data.get('regular_exporter') is True
+def get_is_regular_exporter(answers):
```

Comment:
we could have these in the context processor as well

Reply:
that would be increasing logic in the template. currently, thew view calls `get_is_regular_exporter` to help determine if certain sections of the custom template are to be rendered. We don't want to put logic more complex than `if...endif` in the template if we can help it.